.github/workflows/main.yml

```diff
@@ -35,12 +35,12 @@ jobs:
       - name: Install dependencies
         continue-on-error: ${{ matrix.continue || false }}
         run: bash ./script/install-dependencies.sh
+      - name: Lint
+        continue-on-error: ${{ matrix.continue || false }}
+        run: flake8 --statistics --count
```

Comment:
> --statistics

it is already active, you don't need it here

https://github.com/streamlink/streamlink/blob/8034b1606388e325916baf9f8ce738915fa6b528/setup.cfg#L39